src/client/option.go

```diff
@@ -24,6 +25,13 @@ func WithDatumPutFile(datum string) PutFileOption {
 	}
 }
 
+// WithConcurrency configures the maximum number of tasks in flight created by PutFileURL.
+func WithConcurrency(concurrency int64) PutFileOption {
```

Comment:
`WithConcurrency` might be too overloaded of a term, but I guess this use case got to it first.

`WithImportConcurrency` might be less overloaded for a `PutFileOption`.

Not a blocker.

Reply:
That makes sense, WithConcurrency is overloaded, I agree.